roles/ncn-common/files/scripts/metal/cleanup-live-images.sh

```diff
@@ -26,32 +26,51 @@ set -euo pipefail
 
 ALL=0
 AUTO=0
-BASE="$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/SQFSRAID)"
+INCLUDE_OVERLAYFS=0
+SQUASHFS_DISK=$(grep -Po 'root=[\w=:]+' /proc/cmdline)
+SQUASHFS_BASE="$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/${SQUASHFS_DISK##*=})"
+OVERLAYFS_DISK=$(grep -Po 'rd.live.overlay=[\w=]+' /proc/cmdline)
+OVERLAYFS_BASE="$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/${OVERLAYFS_DISK##*=})"
+
+if [ ! -d ${SQUASHFS_BASE} ] || [ ! -d ${OVERLAYFS_BASE} ]; then
+    echo >&2 "Could not find [$SQUASHFS_BASE] or [$OVERLAYFS_BASE]!"
+    exit 1
+fi
+
 DISK="$(blkid -L SQFSRAID)"
 LIVE_DIR=$(grep -oP 'rd.live.dir=[\w\d-_.]+' /proc/cmdline)
 [ -z "$LIVE_DIR" ] && LIVE_DIR='rd.live.dir=LiveOS'
 LIVE_DIR="${LIVE_DIR#*=}"
 
 function usage {
     cat << EOF
-$0 can be ran with the following:
+$0 will cleanup squashFS and overlayFS that live in the [$SQUASHFS_BASE] and [$OVERLAYFS_BASE] directories.
+
+Without any arguments this script will prompt the user for confirmation before performing any destructive actions.
+
+By default this script will ignore the currently running squashFS.
 
 - Passing any argument other than -y or -a will print this usage.
 - Passing '-y' will automatically clean unused images, bypassing the prompt.
 - Passing '-a' will include all images (unused and used), this will break disk booting unless the node is re-imaged on the next reboot.
+- Passing '-o' will include overlayFS for each image, otherwise these are left alone. This never cleans up the active overlayFS (despite -a being given) because it will break the running operating system.
 EOF
 }
 
-while getopts "ya" opt; do
+while getopts "oya" opt; do
```

Comment:
cosmetic nit... maybe alphabetical order?  and below in the case?  or at least the _same_ order in each.